worker/internal/infrastructure/gcp/gcs.go

```diff
@@ -68,22 +69,17 @@ func (f *fileRepo) Upload(ctx context.Context, name string) (io.WriteCloser, err
 		return nil, gateway.ErrInvalidFile
 	}
 
-	bucket, err := f.bucket(ctx)
+	bucket, err := f.getBucket(ctx)
 	if err != nil {
 		log.Errorf("gcs: upload bucket err: %+v\n", err)
 		return nil, rerror.ErrInternalBy(err)
 	}
 
-	name = filepath.Join(gcsAssetBasePath, name)
+	name = path.Join(gcsAssetBasePath, name)
 
 	object := bucket.Object(name)
-	if err := object.Delete(ctx); err != nil && !errors.Is(err, storage.ErrObjectNotExist) {
-		log.Errorf("gcs: upload delete err: %+v\n", err)
-		return nil, gateway.ErrFailedToUploadFile
-	}
-
 	writer := object.NewWriter(ctx)
-	writer.ObjectAttrs.CacheControl = f.cacheControl
+	// writer.ObjectAttrs.CacheControl = f.cacheControl
```

Comment:
Any specific reason this was commented out?
As it is generally a good practice to set the `Cache-Control` header for objects being uploaded to Google Cloud Storage, or any other web server for that matter.